rmw_opensplice_cpp/CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ endif()
 ament_package()
 
 install(
+  DIRECTORY include/
+  DESTINATION include
+)
```

Comment:
`ament_export_include_directories(..)` will make values available to downstream packages in the variable `pkgname_INCLUDE_DIRS`. A relative path as an argument will be relative to the `CMAKE_INSTALL_PREFIX` and should usually be `include`. An absolute path is well absolute and should not point to the source / build space.

I don't fully understand what you want to install where in this case so I can't give a specific answer what to do here.